### PR TITLE
Fix for TypeError: 'NoneType' object is not iterable

### PR DESCRIPTION
**Pull request by minanagehsalalma**

### CODE REVIEW

File: cloud2butt.py [hunk 1]
```diff
@@ -19,13 +19,12 @@ def getResponseHeadersAndBody(self, content):
   def processHttpMessage(self, tool, is_request, content):
     if is_request:
       return
-    headers, body = self.getResponseHeadersAndBody(content)
```
Comment (everythingishacked) [inline cloud2butt.py:22]: @minanagehsalalma I think this line is still needed? You could then do `if body is not None` below and parse the results.

Comment (minanagehsalalma) [reply cloud2butt.py:22]: Hmm the error is in that line
You can't get headers abd body if the request is empty(none)

Where would it get the needed data from?

It has to check the availability of the data first before trying to fetch to not cause this error

So... M

Comment (minanagehsalalma) [reply cloud2butt.py:22]: Sorry i forgot to the line even that it's in the version of the extension i am using 

I added it now , Hopefully it's now fixed

**Comment by everythingishacked** [comment]
@minanagehsalalma got it, looks like the issue was probably inside the `getResponseHeadersAndBody` function where the `list()` conversion fails on `None` headers. If you get a chance, let me know whether my update on master resolved the issue you saw.

**Comment by minanagehsalalma** [comment]
> @minanagehsalalma got it, looks like the issue was probably inside the `getResponseHeadersAndBody` function where the `list()` conversion fails on `None` headers. If you get a chance, let me know whether my update on master resolved the issue you saw.

@everythingishacked oh thanks it works great again : ) with no errors 
also here a quick addition 

right under these two lines 
```
    body = body.replace(" Cloud", " Butt")
    body = body.replace(" cloud", " butt")
```
add the same lines again but remove the spaces 

```
    body = body.replace("Cloud", " Butt")
    body = body.replace("cloud", " butt")
```
Now it works great on cloudcomputing-news.net : ) i mean **buttcomputing-news.net**

I hope the butt wins : .(
image

**Comment by minanagehsalalma** [comment]
as high as ****

image

**Pull request closed without merging**